sqlmesh/core/macros.py

```diff
@@ -1286,11 +1286,17 @@ def call_macro(
     func: t.Callable,
     dialect: DialectType,
     path: Path,
-    *args: t.Any,
-    **kwargs: t.Any,
+    args: t.Tuple[t.Any, ...],
+    kwargs: t.Dict[str, t.Any],
+    **optional_kwargs: t.Any,
```

Comment:
let's make this a dict also for consistency

Reply:
i don't want to do that because then i'll have to pass in optional_kwargs empty dict even if i don't need it